src/test/scala/TypeCheckerSpec.scala

```diff
@@ -413,13 +413,13 @@ class TypeCheckerSpec extends FunSpec {
       typeCheck("""
               let bucket_idx = 10;
               ---
-              bucket_idx := 20;
+              bucket_idx := (20 as bit<4>);
```

Comment:
You’d probably want to give a larger type to bucket_idx above.

Reply:
Yes, that's not the usual usage of as cast, but I leave it here to remind reader 10 is implicitly viewed as 4 bit number. Another example of usual usage just follows it (let test_var:bit<32> = 10;).